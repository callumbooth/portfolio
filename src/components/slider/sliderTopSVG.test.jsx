import React from 'react';
import renderer from 'react-test-renderer';
import SliderTopSVG from './SliderTopSVG';

it('Svg rotates to 45 deg correctly', () => {
    const component = renderer.create(
        <SliderTopSVG rotation="45" />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

it('Svg rotates by 90 deg correctly', () => {
    const component = renderer.create(
        <SliderTopSVG rotation="90" />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});