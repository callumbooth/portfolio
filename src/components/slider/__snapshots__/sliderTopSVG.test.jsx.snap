// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Svg rotates by 90 deg correctly 1`] = `
<svg
  viewBox="0 0 1520 1080"
  xmlns="http://www.w3.org/2000/svg"
>
  <g>
    <line
      className="a l1"
      transform="translate(1090 0) rotate(90 217.5 217.5)"
      x2={435}
      y2={435}
    />
    <line
      className="a l2"
      transform="translate(417 0) rotate(90 184.5 184.5)"
      x2={369}
      y2={369}
    />
  </g>
</svg>
`;

exports[`Svg rotates to 45 deg correctly 1`] = `
<svg
  viewBox="0 0 1520 1080"
  xmlns="http://www.w3.org/2000/svg"
>
  <g>
    <line
      className="a l1"
      transform="translate(1090 0) rotate(45 217.5 217.5)"
      x2={435}
      y2={435}
    />
    <line
      className="a l2"
      transform="translate(417 0) rotate(45 184.5 184.5)"
      x2={369}
      y2={369}
    />
  </g>
</svg>
`;
